melt_czochralski_Dadzis/base_classes.schema.archive.yaml

```diff
@@ -137,58 +59,17 @@ definitions:
                 # hide: ['children', 'parents']
                 # hide: ['children', 'parents', institute]
                 hide: []
-        geometry:
-          section: '#/Geometry'
+        composition:
+          repeats: true
+          #m_annotations:
+          #  eln:
+          section: '#/EntityAndAmount'
```

Comment:
These Material classes are not yet established, are you using some of them in your schema?

Reply:
I don't think so, I think you told me that I had to include this file in my upload. But maybe that's outdated?

Reply:
Same as previous comment